@types/underscore error TS2322 for `_.chain` after upgrade to v1.9

After upgrade of `@types/underscore` from v1.8.20 to v1.9.0 all usages of `_.chain` in my angular project were broken. For example
```
const arr1: string[] = ['z', 'x', 'y'], query = 'z';
let arr2: string[] = ['a', 'b', 'c'];
...
arr2 = _.chain(arr1)
      .union(arr2)
      .without(query)
      .value()
```
has returned
```
error TS2322: Type 'string' is not assignable to type 'string[]'
```

It seems it was caused by changes done in [this commit](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/9cf5c98aa8c9edc65e70dd078e0f94d72d4aa6d1#diff-bc995fed3367d28165753a2581d1af54).

@regevbr, @DanielRosenwasser was this expected? If yes, what is upgrade path?

--- a/types/underscore/index.d.ts
+++ b/types/underscore/index.d.ts
@@ -5462,7 +5462,7 @@ declare module _ {
         * Wrapped type `any[]`.
         * @see _.without
         **/
-        without(...values: T[]): _Chain<T>;
+        without(...values: T[]): _Chain<T, T[]>;
 
         /**
         * Wrapped type `any[]`.
@@ -5474,7 +5474,7 @@ declare module _ {
         * Wrapped type `any[][]`.
         * @see _.union
         **/
-        union(...arrays: _.List<T>[]): _Chain<T>;
+        union(...arrays: _.List<T>[]): _Chain<T, T[]>;
 
         /**
         * Wrapped type `any[][]`.

--- a/types/underscore/underscore-tests.ts
+++ b/types/underscore/underscore-tests.ts
@@ -572,6 +572,13 @@ function chain_tests() {
     let valuePerYear: number[] = _.chain(yearObject)
         .values()
         .value()
+
+    const arr1: string[] = ['z', 'x', 'y'], query = 'z';
+    let arr2: string[] = ['a', 'b', 'c'];
+    arr2 = _.chain(arr1)
+        .union(arr2)
+        .without(query)
+        .value();
 }
 
 var obj: { [k: string] : number } = {

